spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusMetricsExportAutoConfigurationTests.java

```diff
@@ -313,4 +321,15 @@ public boolean isSampled() {
 
 	}
 
+	@Configuration(proxyBeanMethods = false)
+	@Import(BaseConfiguration.class)
+	static class ExemplarsConfiguration {
```

Comment:
Needed to split `ExemplarsConfiguration` and `SpanConfiguration`, otherwise the test `autoConfiguresExemplarSamplerIfSpanContextSupplierIsPresent` would be falsely asserting (with `.hasSingleBean(ExemplarSampler.class)`) the bean defined here